diff --git a/start.sh b/start.sh
index cc17ce9..4b82f88 100644
--- a/start.sh
+++ b/start.sh
@@ -93,7 +93,7 @@ start() {
     # Get the line number
     startFrom=$(grep -n '########## FTL started' /var/log/pihole/FTL.log | tail -1 | cut -d: -f1)
     # Start the tail from the line number and background it
-    tail --follow=name -n +"${startFrom}" /var/log/pihole/FTL.log &
+    tail --retry --follow=name -n +"${startFrom}" /var/log/pihole/FTL.log &
   else
     echo "  [i] FTL log output is disabled. Remove the Environment variable TAIL_FTL_LOG, or set it to 1 to enable FTL log output."
   fi
